### ISSUE TITLE
Why /ArduCopter/ArduCopter.elf file doesn't exist?

### ISSUE THREAD

**Issue by cnpcshangbo** [ArduPilot/ardupilot_wiki#3928]
Hello,
I am trying to follow this documentation
https://ardupilot.org/dev/docs/using-sitl-for-ardupilot-testing.html#sitl-without-mavproxy-tcp
to run the "elf" file. However, this file doesn't exist in the ArduCopter folder. 
```
bo@bo-Precision-3650-Tower:~/code/ardupilot/ArduCopter$ ls | grep elf           
bo@bo-Precision-3650-Tower:~/code/ardupilot/ArduCopter$ 
```
I was able to run the SITL using the python script. Are there some steps I need to take to generate the elf file or that file is in a different location?
Thanks,
Bo

**Comment by khancyr** [comment]
Hello,

Indeed, the wiki Is not clear on this, the binaries are in build/sitl/bin

**Comment by Hwurzburg** [comment]
@pierre I do not get an .elf file in the ![Screenshot from 2021-12-02 11-30-14](https://user-images.githubusercontent.com/6076534/144473230-27ef4121-c0af-4b71-932c-0775f68fffdd.png)

**Comment by khancyr** [comment]
@Hwurzburg yep, we don't have the .elf and the wiki is also wrong at mentioning that the binary will by in vehicle directory like ./ArduCopter !

**Comment by hendjoshsr71** [comment]
Just for my clarity the only difference between the .elf and the binaries as they are normally is changing the extension to `.elf``?

We do that in the cygwin builds for mission planner. We also use the elf_diff tool on the files as is without the extension in out github actions.

**Comment by Hwurzburg** [comment]
those shown in that Linux directory are the executables I think even without the .elf extension...correct?

**Comment by hendjoshsr71** [comment]
> those shown in that Linux directory are the executables I think even without the .elf extension...correct?

Yes definitely. You can even run them directly. I'm not sure how well doing that is documented on the wiki.

**Comment by Hwurzburg** [comment]
The SITL testing page no longer references a nonexistent `ArduCopter.elf`; it now documents using `sim_vehicle.py --no-mavproxy` with an external GCS.

Feel free to reopen if you feel this has not been addressed.
